wordpress-mobile/WordPress-iOS
# Use Folders for Classes

Removes most of the code from the Xcode project file, instead using a folder.

This makes the file way smaller, should make Xcode way more responsive, and should mean we get far fewer conflicts in the project file.

Uses @mokagio's idea from https://github.com/wordpress-mobile/WordPress-iOS/pull/23679 to make it CocoaPods-compatible.

To Test:

1. Ensure CI passes
2. Ensure that running `bundle exec pod install` locally works
